lisa/trace.py

```diff
@@ -469,104 +1345,372 @@ def cm_func():
                 path,
                 events=events,
                 plat_info=plat_info,
+                # Disable swap if the folder is going to disappear
+                enable_swap=True if filepath else False,
                 **kwargs
             )
 
         proxy.base_trace = trace
 
-    @staticmethod
-    def _process_events(events, proxy_cls):
+    def _get_trace(self, events):
+        logger = self.get_logger()
+        path = self.trace_path
+
+        if self._trace_format is None:
+            if path.endswith('html'):
+                trace_format = 'SySTrace'
+            else:
+                trace_format = 'FTrace'
+        else:
+            trace_format = self._trace_format
+        logger.debug('Parsing {} trace format'.format(trace_format))
+
+        logger.debug('Parsing {} events from {}: {}'.format(trace_format, path, self.events))
+        if trace_format == 'Systems':
```

Comment:
That looks like a rogue autocorrect :)

Reply:
Actually I can just remove the first log, since it's a strict subset of the next one.